VitalHealthConnect/src/main/java/io/tryvital/vitalhealthconnect/VitalHealthConnectManager.kt

```diff
@@ -92,27 +94,59 @@ class VitalHealthConnectManager private constructor(
     private val _status = MutableSharedFlow<SyncStatus>(replay = 1)
     val status: SharedFlow<SyncStatus> = _status
 
+    private val taskScope = CoroutineScope(Dispatchers.Default)
```

Comment:
worth giving it a more meaningful name?

Reply:
This will be the CoroutineScope for launching any tasks not on the main thread, so not sure if there is any more meaningful name